components/NavigationDrawerListItem/NavigationDrawer/index.tsx

```diff
@@ -0,0 +1,116 @@
+import AssignmentOutlinedIcon from '@mui/icons-material/AssignmentOutlined'
+// List component
+import NavigationDrawerListItem from '..'
+// List components from MUI
+import List from '@mui/material/List'
+import ListItemText from '@mui/material/ListItemText'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemButton from '@mui/material/ListItemButton'
+
+interface NavigationDrawerProps {
+  open: boolean
+}
+export default function NavigationDrawer({
+  open = true,
+}: NavigationDrawerProps) {
+  // Params for the Drawer
+  const drawerWidth = 280
+  // Logo definition
+  const logo = (
+    <Image
+      src="/../public/images/CCAHT-Logo.png"
+      alt="CCAHT-logo"
+      height="70"
+      width="239"
+      style={{ objectFit: 'contain' }}
+    />
+  )
+  // state handlers for expandable lists in drawer
+  const [CollapseOpen, setOpen] = React.useState(true)
+  const handleClick = () => {
+    setOpen(!CollapseOpen)
+  }
+  // Drawer setup
+  return open ? (
+    <Box sx={{ display: 'flex' }}>
+      <Drawer
+        variant="permanent"
+        sx={{
+          width: drawerWidth,
+        }}
+      >
+        <Box sx={{ width: drawerWidth }}>
+          <Box
+            sx={{
+              display: 'flex',
+              px: 2,
+              pt: 2,
+              justifyContent: 'center',
+            }}
+          >
+            {logo}
+          </Box>
+          <br />
+          {/* Drawer using ListItem functional component*/}
+          <NavigationDrawerListItem
+            icon={<AssignmentLateOutlinedIcon />}
+            text="Dashboard"
+          />
+          <NavigationDrawerListItem
+            icon={<ArchiveOutlinedIcon />}
+            text="Check In"
+          />
+          <NavigationDrawerListItem
+            icon={<UnarchiveOutlinedIcon />}
+            text="Check Out"
+          />
+          <NavigationDrawerListItem
+            icon={<AssignmentOutlinedIcon />}
+            text="Inventory"
+          />
+          <NavigationDrawerListItem icon={<AccessTimeIcon />} text="History" />
+
+          {/* Collapsable Drawer using ListItem functional component*/}
+          <ListItemButton onClick={handleClick}>
+            <ListItemIcon>
+              <SettingsIcon />
+            </ListItemIcon>
+            <ListItemText primary="Settings" />
+            {CollapseOpen ? <ExpandLess /> : <ExpandMore />}
+          </ListItemButton>
+          <Collapse in={CollapseOpen} timeout="auto" unmountOnExit>
+            {/* <Typography font-size="0.1rem"> */}
+            <List component="div" disablePadding sx={{ pl: 0.8 }}>
+              <NavigationDrawerListItem text="General" collapsable={true} />
+              <NavigationDrawerListItem text="Categories" collapsable={true} />
+              <NavigationDrawerListItem text="Items" collapsable={true} />
+              <NavigationDrawerListItem
+                text="Item Attributes"
+                collapsable={true}
```

Comment:
this and the above instances of `collapsable` do not need to be set to a value, you can simply just put the word `collapsable`. Since it's a boolean, a value of true will be passed to the child for that prop. If the word is not present (like with the other drawer list items), a value of false will be passed to the child for the `collapsable` prop